package.json

```
@@ -36,18 +38,21 @@
   },
   "dependencies": {
     "@types/supertest": "2.0.8",
-    "express": "4.17.1"
+    "express": "4.17.1",
+    "swagger-ui-express": "^4.1.2",
```

Could you please target the exact version as follow? this to avoid an update to the `package-lock.json` locally by a dev if a new version comes and therefore avoid surprises due to incompatibility or security vulnerabilities
```suggestion
    "swagger-ui-express": "4.1.2",
```

---

done!